js/main.js

```diff
@@ -1,44 +1,58 @@
-function copyMatsList() { // copies matsList to clipboard
+function selectOption() {
+    let select = document.getElementById("dropdown-select");
+    let recipe = '';
+    if (select.value === 'none') {
+        recipe = '';

```

Comment:
If all you are going to do here is set it to an empty string, that is a bit of a waste of an if (seeing as the line above does the exact same thing.

You have 2 options as I see it:
1. Change this line to: `document.getElementById("recipebox").innerHTML = recipe;` and then put a return. Then you can just remove the else.
2. Invert the if condition, and move the for loop up. Then you can just delete the else and the behaviour will be the same